extensions/security/deployment/src/main/java/io/quarkus/security/deployment/AdditionalRolesAllowedTransformer.java

```diff
@@ -11,27 +11,28 @@
 import org.jboss.jandex.AnnotationValue;
 
 import io.quarkus.arc.processor.AnnotationsTransformer;
+import io.quarkus.gizmo.MethodDescriptor;
 
 public class AdditionalRolesAllowedTransformer implements AnnotationsTransformer {
 
-    private final Set<String> classNames;
+    private final Set<MethodDescriptor> methods;
```

Comment:
Again, I would much rather prefer us to use something other than a Gizmo type here

Reply:
I'll have look on Monday

Reply:
Thanks, well appreciated.